src/components/block/index.ts

```diff
@@ -448,8 +461,20 @@ export default class Block extends EventsDispatcher<BlockEvents> {
   public set selected(state: boolean) {
     if (state) {
       this.holder.classList.add(Block.CSS.selected);
+
+      const range = SelectionUtils.range;
```

Comment:
move it to a separate method. Should it be inside the Block class? Maybe it is related to the Caret util

Reply:
Agree would be better to place it to Caret util but it is not util yet and Block class doesn't have access to it